From e5186e003e46c2a8d56b6c274ad4259840b9b48a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hubert=20Figui=C3=A8re?= <hub@figuiere.net>
Date: Fri, 24 Dec 2021 15:42:17 -0500
Subject: [PATCH] __DT_CLONE_TARGETS__ was undefined on aarch64

- it's probably broken on armv7 too and other non intel arch
---
 src/common/darktable.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/common/darktable.h b/src/common/darktable.h
index b841bf2bc15..aad6169f8b6 100644
--- a/src/common/darktable.h
+++ b/src/common/darktable.h
@@ -133,6 +133,8 @@ typedef unsigned int u_int;
 # elif defined(__PPC64__)
 /* __PPC64__ is the only macro tested for in is_supported_platform.h, other macros would fail there anyway. */
 #define __DT_CLONE_TARGETS__ __attribute__((target_clones("default","cpu=power9")))
+# else
+#define __DT_CLONE_TARGETS__
 # endif
 #else
 #define __DT_CLONE_TARGETS__
